api/src/certification/scoring/domain/models/CapacitySimulator.js

```diff
@@ -31,3 +48,24 @@ export class CapacitySimulator {
     });
   }
 }
+
+export function findIntervalIndex({ score, weightsAndCoefficients, scoringIntervalsLength }) {
```

Comment:
Cela ne fait-il pas doublon avec `Intervals.findIntervalIndex` ? Quelle est la différence entre les deux ?

Reply:
Proposition de renommage d'un des arguments

```suggestion
export function findIntervalIndex({ score, weightsAndCoefficients, maxNumberOfInterval }) {
```

Reply:
Les deux fonctions sont différentes.
Dans le cadre de la méthode disponible sur la classe `Intervals` il est uniquement possible de retrouver l'index de la maille à partir d'une capacité (ce de quoi les mailles sont constituées)
Dans le cas de cette fonction, on cherche à obtenir l'index de la maille à partir d'un score, ce qu'il n'est possible de faire que via l'algorithme ci-dessous et que l'on ne cherche à réaliser que dans le cadre du simulateur de capacité (score -> capacité)